test/utils/deploy/deploy.go

```diff
@@ -133,20 +137,23 @@ func DeployNROScheduler() *nropv1.NUMAResourcesScheduler {
 		klog.Infof("condition: %v", cond)
 
 		return cond.Status == metav1.ConditionTrue
-	}).WithTimeout(5*time.Minute).WithPolling(10*time.Second).Should(BeTrue(), "NRO Scheduler condition did not become available")
+	}).WithTimeout(timeout).WithPolling(NROSchedulerPollingInterval).Should(BeTrue(), "NRO Scheduler condition did not become available")
 	return nroSchedObj
 }
 
-func TeardownNROScheduler(nroSched *nropv1.NUMAResourcesScheduler, timeout time.Duration) {
+func TeardownNROScheduler(ctx context.Context, nroSched *nropv1.NUMAResourcesScheduler, timeout time.Duration) {
 	GinkgoHelper()
 
-	if nroSched != nil {
-		err := e2eclient.Client.Delete(context.TODO(), nroSched)
-		Expect(err).ToNot(HaveOccurred())
-
-		err = wait.With(e2eclient.Client).Interval(10*time.Second).Timeout(timeout).ForNUMAResourcesSchedulerDeleted(context.TODO(), nroSched)
-		Expect(err).ToNot(HaveOccurred(), "NROScheduler %q failed to be deleted", nroSched.Name)
+	if nroSched == nil {
+		// nothing to do!
+		return
 	}
+
+	err := e2eclient.Client.Delete(ctx, nroSched)
```

Comment:
`Expect(e2eclient.Client.Delete(ctx, nroSched)).To(Succeed())`

Reply:
thanks, done